torch/backends/cuda/__init__.py

```diff
@@ -313,6 +313,24 @@ def math_sdp_enabled():
     return torch._C._get_math_sdp_enabled()
 
 
+def enable_math_sdp(enabled: bool):
+    r"""
+    .. warning:: This flag is beta and subject to change.
+
+    Enables or disables fp16/bf16 reduction in math scaled dot product attention.
+    """
+    torch._C.set_math_sdp_allow_fp16_bf16_reduction(enabled)
+
+
+def math_sdp_enabled():
```

Comment:
You are clashing with the existing functions, an you use a different name more specific to what this is doing